doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
-*pre-save*::
-  Executed before the data files are loaded.
-*post-save*::
-  Executed after the data files are saved.
-
-Some examples can be found in the `contrib/hooks/` directory of the calcurse
-source tree.
+If the calcurse data files contain many entries which are no longer needed or
+wanted, they can, of course, be removed interactively. If there are many, it
+can be a tedious task and may be done better as in the following two examples.
+'Note.' The filter options specify what is retained, not what is removed!
+Presently it is not possible to list the items to be removed.
 
-Notes
------
+`calcurse --input-datefmt 4 -G --filter-start-before 2015-1-1`::
+  List event and appointment entries in the data files with a start time
+  before 1 January 2015, and all TODO entries.
++
+*Purge*. When +-G+ is replaced by +-P+, those entries are removed. This may
+remove recurring items that have occurrences after 1 January 2015.
 
-Calcurse interface contains three different panels (calendar, appointment list,
-and todo list) on which you can perform different actions. All the possible
-actions, together with their associated keystrokes, are listed on the status
-bar. This status bar takes place at the bottom of the screen.
+`calcurse --input-datefmt 1 -G --filter-start-from 11/1/2015 --filter-type event,apt`::
+  List (ordinary) event and appointment entries with a start time of 1
+  November 2015 or later.
 
-At any time, the built-in help system can be invoked by pressing the '?' key.
-Once viewing the help screens, informations on a specific command can be
-accessed by pressing the keystroke corresponding to that command.
+`calcurse -G --filter-type apt --format-apt "%(hash) %m\n"`::
+  For each appointment list the SHA1 hash of the data file entry followed by
+  the description.
 
-Configuration
--------------
+`calcurse -G --filter-type apt --format-apt "%(duration:%d/%EH/%EM)\t%m\n"`::
+  For each appointment list the (total) duration as either days, hours or
+  minutes followed by the description.
 
-The calcurse options can be changed from the configuration menu (shown when 'C'
-is hit). Five possible categories are to be chosen from : the color scheme, the
-layout (the location of the three panels on the screen), notification options,
-key bindings configuration menu, and more general options (such as automatic
-save before quitting).  All of these options are detailed in the configuration
-menu.
+`calcurse -G --filter-type apt --format-apt "%(start:%c) %(duration:%d %H:%M)\t%m\n"`::
+  For each appointment list the start time in a localized standard format,
+  followed by the duration in days, hours and minutes, followed by the
+  description.
 
-Files
+[[_files]]
+FILES
 -----
 
-The following structure is created in your $HOME directory (or in the directory
-you specified with the *-D* option), the first time calcurse is run:
+The following structure is created by default in your $HOME directory the
```

Comment:
Maybe we can change "$HOME directory" to "home directory" while touching these lines...